lib/src/importer-registry.ts

```diff
@@ -11,23 +11,23 @@ import {FileImporter, Importer, Options} from './vendor/sass';
 import * as proto from './vendor/embedded_sass_pb';
 import {catchOr, thenOr, PromiseOr} from './utils';
 
-const entryPointPathKey = Symbol();
+const entryPointDirectoryKey = Symbol();
 
 export class NodePackageImporter {
-  [entryPointPathKey]?: string;
+  [entryPointDirectoryKey]?: string;
 
-  constructor(entryPointPath?: string) {
-    entryPointPath = entryPointPath
-      ? p.resolve(process.cwd(), entryPointPath)
+  constructor(entryPointDirectory?: string) {
+    entryPointDirectory = entryPointDirectory
+      ? p.resolve(process.cwd(), entryPointDirectory)
       : require.main?.filename;
```

Comment:
@jamesnw Does this need to be `p.dirname(require.main?.filename)`?

Reply:
Good catch- fixed in https://github.com/sass/embedded-host-node/pull/260/commits/be27f0b1987c695d7f08bb4fc9d429bc8660fbfa